### PR TITLE
userAgent Bug

### PR DESCRIPTION
**Pull request by echatzief**
#### What does this PR address?
- [ ] Gitub issue (Add reference - #XX)
- [ ] Refactoring
- [ ] New feature
- [X] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [X] Yes
- [ ] No

### CODE REVIEW

**Comment by patrick-tolosa** [comment]
Hey @echatzief - What bug does this PR fix?

**Comment by echatzief** [comment]
> Hey @echatzief - What bug does this PR fix?

The module is used at the strapi-provider-upload-cloudinary and the navigator is globally defined. Thus, using the library with node.js the navigator breaks (because it's not defined) and throws error to the above plugin. So, you must define the userAgent inside each function not globally.

**Comment by patrick-tolosa** [comment]
Thanks, sounds like the right approach.

We'd like to avoid this code duplication though, can you please move the navigator line to a function? `getNavigator()` (or any other suitable name) instead of copy pasting this line over and over.

**Comment by patrick-tolosa** [comment]
Almost done here, please move it to the top of the file and we're good to merge.

**Review by patrick-tolosa — APPROVED** [review]

File: src/util/browser.js [hunk 1]
```diff
@@ -3,31 +3,39 @@
  * https://github.com/videojs/video.js/blob/4238f5c1d88890547153e7e1de7bd0d1d8e0b236/src/js/utils/browser.js
  */
 
-// User agent example taken from Safari desktop:
-const useAgent = navigator && navigator.userAgent || '';
+/**
+* Retrieve from the navigator the user agent property.
+* @returns user agent property.
+*/
+function getNavigator(){
```
Comment (maoznir) [inline src/util/browser.js:10]: @echatzief Can you please change this to getUserAgent() ?

**Review by maoznir — CHANGES_REQUESTED** [review]

File: src/util/browser.js (hunk 1 above)
Comment (echatzief) [reply src/util/browser.js]: Sure. I changed it

**Review by maoznir — APPROVED** [review]

**Pull request merged**